src/components/contact-footer.astro

```diff
@@ -0,0 +1,88 @@
+---
+import ContactForm from '@components/contact-form.astro';
+---
+
+<section class="contact-footer">
+  <div class="container">
+    <img src="/assets/about-bjerk.jpg" alt="Bilde av Bjerk" />
+    <section class="content">
+        <h2>Vi vil gjerne høre fra deg!</h2>
+        <p>
+          Uansett om du har en idé som vil forandre verden, eller om det er en
+          liten sak som irriterer deg i hverdagen, så er vi like ivrige.
```

Comment:
`liten sak som irriterer deg` was suggested to get a change. The word `irriterer`is negatively charged. I can't remember exact word that was sought changed, but how about something like:

```suggestion
          Uansett om du har en idé som vil forandre verden, eller om det er en
          liten sak i hverdagen som må fikses, så er vi like ivrige.
```